### PR TITLE
Add bicycle having probability

### PR DESCRIPTION
**Pull request by SHaaD94**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2849)

### CODE REVIEW

**Comment by SHaaD94** [comment]
Test!

File: src/main/resources/beam-template.conf [hunk 1]
```diff
@@ -225,6 +225,7 @@ beam.agentsim.agents.households.inputHouseholdAttributesFilePath = ${beam.inputD
 beam.agentsim.agents.vehicles.fractionOfInitialVehicleFleet = "double | 1.0"
 beam.agentsim.agents.vehicles.vehicleAdjustmentMethod = "UNIFORM"
 beam.agentsim.agents.vehicles.downsamplingMethod = "SECONDARY_VEHICLES_FIRST"
+beam.agentsim.agents.vehicles.bicycleHavingProbability = "double | 1.0"
```
Comment (wrashid) [inline src/main/resources/beam-template.conf:228]: fractionOfPeopleWithBicycle

File: src/main/scala/beam/utils/scenario/UrbanSimScenarioLoader.scala [hunk 2]
```diff
@@ -161,13 +165,10 @@ class UrbanSimScenarioLoader(
           )
           .toBuffer
 
-        beamScenario.vehicleTypes.values
-          .find(_.vehicleCategory == VehicleCategory.Bike) match {
-          case Some(vehType) =>
-            vehicleTypes.append(vehType)
-          case None =>
-            throw new RuntimeException("Bike not found in vehicle types.")
+        if (rand.nextDouble() <= beamScenario.beamConfig.beam.agentsim.agents.vehicles.bicycleHavingProbability) {
```
Comment (wrashid) [inline src/main/scala/beam/utils/scenario/UrbanSimScenarioLoader.scala:168]: add simple test: make bike attractive (increase bike intercept) and compare scenario with 0% bike vs. 100%

**Comment by REASY** [comment]
test!

File: src/main/resources/beam-template.conf (hunk 1 above)
Comment (SHaaD94) [reply src/main/resources/beam-template.conf:228]: fixed

**Comment by SHaaD94** [comment]
Test!

File: src/main/scala/beam/utils/scenario/UrbanSimScenarioLoader.scala (hunk 2 above)
Comment (SHaaD94) [reply src/main/scala/beam/utils/scenario/UrbanSimScenarioLoader.scala]: Here are the results, fraction equals to 100% (1.0)
image
Fraction equals to 0.0:
image
Fraction equals to 0.5:
image

**Pull request merged**